app/gateway/api/transfer/schema/create_request.go

```diff
@@ -7,7 +7,7 @@ import (
 
 type CreateRequest struct {
 	// DestinationID is the ID of the transfer that wil receive the money
-	DestinationID string `json:"destination_id" example:"123e4567-e89b-12d3-a456-426614174000"`
+	DestinationID string `json:"account_destination_id" example:"123e4567-e89b-12d3-a456-426614174000"`
```

Comment:
```suggestion
	DestinationID string `json:"destination_account_id" example:"123e4567-e89b-12d3-a456-426614174000"`
```
Tem uma inversão aqui, a gente inverte a posição do sujeito e adjetivo no inglês. Carro azul é blue car por exemplo, e no teu exemplo conta destino vira destination account

Reply:
Regis, nesse caso nem é questão do inglês, mas sim dos requisitos do teste:
/transfers

A entidade Transfer possui os seguintes atributos:

    id
    account_origin_id
    account_destination_id
    amount
    created_at

Reply:
Tbm tem isso, não lembrava dos requisitos :v

Reply:
O ponto é justamente o levantado pelo @VitorinoAssuncao. Essas tags vêm da especificação do desafio.